server/game/cards/09.3-TSs/GrimoiresAndMore.js

```diff
@@ -0,0 +1,50 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class GrimoiresAndMore extends DeedCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            title: 'React: Grimoires & More',
+            when: {
+                onCardEntersPlay: event => event.card.controller === this.controller && event.card.getType() === 'dude' && event.card.hasKeyword('huckster')
+            },
+            cost: ability.costs.bootSelf(),
+            handler: context => {
+                const dude = context.event.card;
+                context.player.pull((pulledCard) => {
+                    if('clubs' === pulledCard.suit.toLowerCase()) {
```

Comment:
I would rather use `pulledSuit` parameter from the pull function. That is because if there will be in the future some card that would replace the pull effect and maybe change suit, the new suit will be passed as `pulledSuit` while the `pulledCard` will have the default suit. (e.g. `AnalyticalCognisizer` for gadgets)
```suggestion
                context.player.pull((pulledCard, pulledValue, pulledSuit) => {
                    if('clubs' === pulledSuit.toLowerCase()) {
```